pkgs/by-name/sw/switch-to-configuration-ng/package.nix

```diff
@@ -10,17 +10,9 @@ rustPlatform.buildRustPackage {
   pname = "switch-to-configuration";
   version = "0.1.0";
 
-  src = lib.fileset.toSource {
-    root = ./.;
-    fileset = lib.fileset.unions [
-      ./Cargo.lock
-      ./Cargo.toml
-      ./build.rs
-      ./src
-    ];
-  };
+  src = ./src;
```

Comment:
Non‐blocking nit: `./switch-to-configuration` would match the `cargo new` naming convention.